app/main.py

```diff
@@ -1,6 +1,16 @@
-def main():
-    # write your code here
-    pass
+def main() -> None:
+    file_name = input("Enter name of the file: ")
+    content = []
+
+    while True:
+        new_line = input("Enter new line of content: ")
+        if new_line == "stop":
```

Comment:
Add newline at the end of file, so when u edit existing file it will be working correct